python3/plugins/extauth-hook-AD.py

```diff
@@ -61,17 +61,14 @@ def setup_logger():
 logger = logging.getLogger(__name__)
 
 
-def run_cmd(cmd, log_cmd=True):
-    """Helper function to run command"""
+def run_cmd(command: "list[str]"):
```

Comment:
IIRC, the log_cmd argument is by-intention,
some command contains user credentials, which should NOT be logged.